### ISSUE TITLE
Kafka 0.8.0 installation issue

### ISSUE THREAD

**Issue by airtonjal** [Homebrew/homebrew-versions#1260]
Hi

when I try to install Kafka 0.8.0:

```brew install -v homebrew/versions/kafka080```

I get the following error:

```Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=512m; support was removed in 8.0
error: error while loading CharSequence, class file '/Library/Java/JavaVirtualMachines/jdk1.8.0_66.jdk/Contents/Home/jre/lib/rt.jar(java/lang/CharSequence.class)' is broken```

It seems that [Kafka 0.8.0](https://github.com/apache/kafka/tree/0.8.0) uses Scala 2.9.x, which is [not compatible with Java 8](http://www.scala-lang.org/download/2.9.3.html).

kafka080.rb formula contains:
```ruby
  depends_on :java => "1.7+"`
```
I'm stuck with this, any help is deeply appreciated.

Thanks!

**Comment by mikemcquaid** [comment]
> depends_on :java => "1.7+"

It seems this should probably be changed to `depends_on :java => "1.7"` and will work if you install a version of JDK 7.

**Comment by apjanke** [comment]
That fix worked for me locally (along with adding `ENV.java_cache` to fix sandbox-related `sbt` errors). Put in a PR for it: #1346.

**Comment by apjanke** [comment]
Resolved by #1346.
